加载数据...
['a stirring , funny and finally transporting re-imagining of beauty and the beast and 1930s horror films', 'apparently reassembled from the cutting-room floor of any given daytime soap .', "they presume their audience wo n't sit still for a sociology lesson , however entertainingly presented , so they trot out the conventional science-fiction elements of bug-eyed monsters and futuristic women in skimpy clothes .", 'this is a visually stunning rumination on love , memory , history and the war between art and commerce .', "jonathan parker 's bartleby should have been the be-all-end-all of the modern-office anomie films ."]
[1, 0, 0, 1, 1]
Time usage: 0:00:07
Some weights of the model checkpoint at /home/huyiwen/pretrained/bert were not used when initializing BertModel: ['cls.predictions.transform.LayerNorm.weight', 'cls.predictions.decoder.weight', 'cls.seq_relationship.bias', 'cls.predictions.transform.dense.weight', 'cls.seq_relationship.weight', 'cls.predictions.bias', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.bias']
- This IS expected if you are initializing BertModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Traceback (most recent call last):
  File "/home/huyiwen/CV/bilstm/distill.py", line 68, in <module>
    S_model = biLSTM(cfg).to(cfg.device)
  File "/home/huyiwen/CV/bilstm/models/lstm_mpo.py", line 177, in __init__
    self.fc2 = LinearDecomMPO(config.FC_dim, config.num_classes, *self.fc2_mpo_config)
  File "/home/huyiwen/CV/bilstm/models/linear_mpo.py", line 380, in __init__
    self.tensor_set =  self._parameter_decompose(weight=weight, device=device, dtype=dtype)
  File "/home/huyiwen/CV/bilstm/models/linear_mpo.py", line 395, in _parameter_decompose
    tensor_set, _, _ = self.mpo.matrix2mpo(weight)
  File "/home/huyiwen/CV/bilstm/models/linear_mpo.py", line 258, in matrix2mpo
    tensor_set = self.get_tensor_set(inp_matrix)
  File "/home/huyiwen/CV/bilstm/models/linear_mpo.py", line 128, in get_tensor_set
    res = res.reshape(tuple(self.mpo_input_shape[:]) + tuple(self.mpo_output_shape[:]))
ValueError: cannot reshape array of size 384 into shape (6,2,1,2,8,5,1,1,1,1)